### DISCUSSION TITLE
ArgError: option requires argument: --store

### DISCUSSION THREAD

**Discussion by racsou** [swellstores/community#258] [Help] [answered]
I got this problem when i tried this command : npm install

file:///var/www/html/swell/horizons/node_modules/@swell/cli/dist/bin/swell-2acb6c0f.js:21659
                                                throw new ArgError(
                                                      ^

ArgError: option requires argument: --store
    at arg (file:///var/www/html/swell/horizons/node_modules/@swell/cli/dist/bin/swell-2acb6c0f.js:21659:13)
    at file:///var/www/html/swell/horizons/node_modules/@swell/cli/dist/bin/swell-2acb6c0f.js:29597:14
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:526:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'ARG_MISSING_REQUIRED_LONGARG'
}
npm ERR! code 1
npm ERR! path /var/www/html/swell/horizons
npm ERR! command failed
npm ERR! command sh -c -- husky install && if [ -f codegen.yml ] && [ -f build-utils/generate-theme.mjs ]; then npm run setup; fi

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/racsou/.npm/_logs/2023-05-26T19_08_04_379Z-debug-0.log

**Comment by sags95** [comment, marked as answer]
@racsou We're preparing a fix that includes lots of changes to the way Horizon is installed. In the meantime you can use this branch https://github.com/swellstores/horizon/tree/fix/swell-cli-sdv2.

The main difference is you'll also have to declare a `SWELL_STOREFRONT_ID` env var as described in the README
